src/fallbacks.jl

```diff
@@ -252,7 +255,9 @@ getcolumn(x::CopiedColumns, nm::Symbol) = getcolumn(source(x), nm)
 columnnames(x::CopiedColumns) = columnnames(source(x))
 
 # here's our generic fallback Tables.columns definition
-@inline function columns(x::T) where {T}
+columns(x::T) where {T} = _columns(x)
```

Comment:
```suggestion
@inline columns(x::T) where {T} = _columns(x)
```
Keep the inline annotation?